pandas/tests/scalar/test_parsing.py

```diff
@@ -136,4 +136,4 @@ def test_try_parse_dates(self):
 
         result = parsing.try_parse_dates(arr, dayfirst=True)
         expected = [parse(d, dayfirst=True) for d in arr]
-        assert np.array_equal(result, expected)
+        assert tm.assert_numpy_array_equal(result, expected)
```

Comment:
Remove the `assert` keyword.  `tm.assert_...` takes care of that for you.  As written, this will cause an `AssertionError`.